web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -144,28 +181,29 @@ internal fun Route.behandlingRoutes(
     authorize(Brukerrolle.Saksbehandler) {
         post("$behandlingPath/{behandlingId}/beregn") {
             call.withBehandlingId { behandlingId ->
+                val behandling = søknadsbehandlingService.hent(HentRequest(behandlingId)).getOrHandle {
+                    return@withBehandlingId call.svar(NotFound.message("Fant ikke behandling"))
+                }
                 call.withBody<NyBeregningForSøknadsbehandlingJson> { body ->
-                    body.toDomain(behandlingId, Saksbehandler(call.suUserContext.navIdent))
+                    body.toDomain(behandlingId, Saksbehandler(call.suUserContext.navIdent), Stønadsperiode.create(behandling.periode))
```

Comment:
Hadde håpet å unngå å sende inn denne perioden her, da behandlingen selv er i stand til å finne ut dette "litt lenger inn". Oppførselen i bildet for fradrag/deler av periode oppførte seg såpass underlig at jeg var sikker på at jeg hadde knekt noe og revertet derfor disse endringene. I ettertid virker det som at oppførselen fortsatt er like underlig, men jeg orket ikke å endre det tilbake tilbake.

Reply:
Jeg og Richard satt og så på revurdering og beregnOgSimuler-endepunktet og observerte noe av det samme. Vi sender inn perioden til endepunktet selvom den allerede finnes fra opprettelsen av.

Reply:
Det fremstår litt uheldig.